CrewChiefV4/iRacing/Sim.cs

```diff
@@ -91,19 +71,12 @@ private void GetDrivers(SessionInfo info)
                 {
                     driver = Driver.FromSessionInfo(info, id);
 
-                    // If no driver found, end of list reached
                     if (driver == null)
                     {
                         continue;
                     }
 
                     driver.IsCurrentDriver = false;
```

Comment:
now we're including the pace car in the array, are you confident it doesn't trigger weird messages?

Reply:
Yes it should be ruled out in the opponent array